### PR TITLE
[MRG] X-NES

### PR DESCRIPTION
**Pull request by AlexanderFabisch**

### CODE REVIEW

File: bolero/optimizer/nes.py [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+from .optimizer import Optimizer
+from ..utils.validation import check_random_state, check_feedback
+from ..utils.log import get_logger
+
+
+class XNESOptimizer(Optimizer):
+    """Exponential Natural Evolution Strategies (xNES).
+
+    See `Wikipedia <http://en.wikipedia.org/wiki/Natural_evolution_strategy>`_
+    for details.
```
Comment (AlexanderFabisch) [inline bolero/optimizer/nes.py]: Link JMLR paper

**Comment by AlexanderFabisch** [comment]
@maotto Could you take a look at this merge request? (check unit tests, documentation, example, etc.)

CI seems to fail currently because of a bug in travis ci.

**Comment by maotto** [comment]
> @maotto Could you take a look at this merge request? (check unit tests, documentation, example, etc.)

I can take a look next week if it is still needed.

**Comment by AlexanderFabisch** [comment]
I don't think so, thanks anyway!

**Pull request merged**